index.js

```diff
@@ -1,10 +1,9 @@
 
 export { fetchingData } from './src/artemisa/component'
-import { artemisa } from './src/artemisa/reducer'
-import { dataService } from './src/core/service'
+export { artemisa as reducer, isFetching, isFetched, processApiAction } from './src/artemisa/reducer'
```

Comment:
This may be out of the scope of this PR so feel free to take this to an issue so we can discuss better, but I think we should export something else than `reducer`, given that name is too generic and could potentially collide with something a user of this library could already have in their code. Don't have a strong suggestion or preference on what to replace this with, maybe `artemisaReducer` to keep it clear what it is?